server/tests/unit/__init__.py

```diff
@@ -3,43 +3,23 @@
 
 from os import listdir, path
 
-import pandas as pd
 from flask_compress import Compress
 from flask_cors import CORS
 
 from server.common.config.app_config import AppConfig
-from server.common.fbs.matrix import encode_matrix_fbs
 from server.app.app import Server
 from server.tests import FIXTURES_ROOT
 
 
-def make_fbs(data):
-    df = pd.DataFrame(data)
-    return encode_matrix_fbs(matrix=df, row_idx=None, col_idx=df.columns)
-
-
-def skip_if(condition, reason: str):
-    def decorator(f):
-        def wraps(self, *args, **kwargs):
-            if condition(self):
-                self.skipTest(reason)
-            else:
-                f(self, *args, **kwargs)
-
-        return wraps
-
-    return decorator
-
-
-def app_config(data_locator, extra_server_config={}, extra_dataset_config={}):
+def app_config(extra_server_config=None, extra_dataset_config=None):
+    extra_server_config = extra_server_config if extra_server_config else {}
+    extra_dataset_config = extra_dataset_config if extra_dataset_config else {}
```

Comment:
```suggestion
    extra_dataset_config = extra_dataset_config or {}
```

Reply:
curious why these new lines are now needed?

Reply:
those parameters were default set to `{}` which is mutable and could cause strange behavior if manipulated. I changed the defaults to `None` which is immutable, and set them to `{}` if `None`